src/modules/button/button.tsx

```diff
@@ -208,6 +209,7 @@ const Button: React.FC<IProps> = React.memo(
         {/* Label */}
         <StyledLabel
           $font={font}
+          accessibilityLabel="button-label"
```

Comment:
My only concern is that if there are say 5 buttons on the page, automation will not know which one to pick.